android/build.gradle

```diff
@@ -69,9 +65,6 @@ android {
     }
     flavorDimensions "dependencyVersions"
     productFlavors {
-        unityIAP {
-            dimension "dependencyVersions"
-        }
         latestDependencies {
             dimension "dependencyVersions"
         }
```

Comment:
I guess since we only have one flavor with this, we could also remove this, but since we plan to bring it back, I think it's probably ok to leave it.